docs/wiki/Home.md

```diff
@@ -18,6 +18,9 @@ If you're unfamiliar with Infrastructure as Code or wonder how you can use the c
   - [**Scenario 2:** Onboard module library and CI environment](./Getting%20started%20-%20Scenario%202%20Onboard%20module%20library%20and%20CI%20environment)
     - [Check namePrefix availability](./Getting%20started%20-%20Check%20NamePrefix%20availability)
   - [**Scenario 3:** Contribute](./Getting%20started%20-%20Scenario%203%20Contribute)
+- [Fetching latest changes](./Fetching%20latest%20updates)
```

Comment:
Decide between `changes` and `updates` as a title for the whole section and keep the same in all titles and filenames. E.g., the title of this page is `Fetching latest changes`, but the corresponding .md filename is `Fetching latest updates`.

Reply:
renamed file and updated links in home and sidebar